File: Algorithm-CPP/63_인접행렬.cpp
#include <iostream>
using namespace std;

int main() {
    int n, m;
    cin >> n >> m;
    int arr[n][n];
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            arr[i][j] = 0;
        }
    }

    for(int i = 0; i < m; i++) {
        int row, col, cost;
        cin >> row >> col >> cost;
        arr[row - 1][col - 1] = cost;
    }

    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            cout << arr[i][j] << " ";
        }
        cout << endl;
    }
}

File: Algorithm-Swift/Algorithm-Swift/Inflearn/63_인접행렬.swift
//
//  63_인접행렬.swift
//  Algorithm-Swift
//
//  Created by SHIN YOON AH on 2023/09/02.
//

import Foundation

func solution63() {
    let nm = readLine()!.split(separator: " ").map { Int($0)! }, n = nm.first!, m = nm.last!
    var arr = Array(repeating: Array(repeating: 0, count: n), count: n)
    for _ in 0..<m {
        let input = readLine()!.split(separator: " ").map { Int($0)! }
        let row = input.first!, col = input[1], cost = input.last!
        arr[row - 1][col - 1] = cost
    }

    arr.forEach {
        $0.forEach { print($0, terminator: " ") }
        print("")
    }
}
